### DISCUSSION TITLE
[Q&A] Can we set environment in Administration Mode using pac cli?

### DISCUSSION THREAD

**Discussion by sanjeev40084** [microsoft/powerplatform-vscode#616] [Q&A]
Are there any pac org command that allows to put environment in Administration mode? i ran `pac org list-settings` command and didn't see anything.

**Comment by petrochuk** [comment]
At this point we don't have it.

**Reply by sanjeev40084** [reply]
@petrochuk shall i submit as new idea?

**Reply by petrochuk** [reply]
Any ideas are welcome. We will be looking at most upvoted ones
